models/current_knesset_member.py

```diff
@@ -18,18 +18,14 @@ class KnessetMemberByIndividual(BaseModel):
                                           "ועדת המשנה לקידום תעשיית ההייטק (יו\"ר ועדה)"]
     year_total_hours_attended: str = "[2010-86,2011-219,2012-185,2013-1252,2014-270,2015-699,2016-586,2019-437,2020-688,2021-434,2022-222,2023-818]" 
 
-class KnessetMemberResponseByIndividual(BaseModel):
-    success: bool
-    data: KnessetMemberByIndividual
-    
 class KnessetMemberByPersonal(BaseModel):
-    personal_id: int = 427
+    PersonID: int = 427

```

Comment:
I took the field names as they are shown in the adjusting tables in the database. Sometimes, they are displayed in PascalCase, and sometimes in snake_case. Do you think it's still better to change all fields to snake_case? Wouldn't that be confusing? Or it will be clear?